CHANGELOG.md

```diff
@@ -24,6 +24,7 @@
 * [ENHANCEMENT] Distributor: reduce CPU and memory when an high number of errors are returned by the distributor on the write path. #3990
 * [ENHANCEMENT] Put metric before label value in the "label value too long" error message. #4018
 * [ENHANCEMENT] Allow use of `y|w|d` suffixes for duration related limits and per-tenant limits. #4044
+* [ENHANCEMENT] Use enums to represent client types in factory functions. #4048
```

Comment:
**nit:** this change shouldn't have any behavioral changes for the user so there is no need for a changelog entry.

```suggestion
```